src/js/views/patients/patient/sidebar/sidebar_views.js

```diff
@@ -1,142 +1,136 @@
 import _ from 'underscore';
 import anime from 'animejs';
 import moment from 'moment';
+import Radio from 'backbone.radio';
 
 import hbs from 'handlebars-inline-precompile';
-import { View, CollectionView, Region } from 'marionette';
+import { View, CollectionView } from 'marionette';
 
-import intl, { renderTemplate } from 'js/i18n';
 import ModelRender from 'js/behaviors/model-render';
 
-import PatientSidebarTemplate from './patient-sidebar.hbs';
-import PatientSidebarGroupsTemplate from './patient-sidebar-groups.hbs';
-
 import './patient-sidebar.scss';
 import 'sass/domain/engagement-status.scss';
 
-const i18n = intl.patients.patient.sidebar.sidebarViews;
-
-const patientSet = [
-  {
-    heading: i18n.dobLabel,
+const sidebarWidgets = {
+  dob: {
     template: hbs`{{formatHTMLMessage (intlGet "patients.patient.sidebar.sidebarViews.dob") dob=(formatMoment dob "LONG" inputFormat="YYYY-MM-DD") age=age}}`,
-    data(patient) {
-      const dob = patient.get('birth_date');
+    templateContext() {
+      const dob = this.model.get('birth_date');
 
       const age = moment().diff(moment(dob, 'YYYY-MM-DD'), 'years');
 
       return { dob, age };
     },
   },
-  {
-    heading: i18n.sexLabel,
+  sex: {
     template: hbs`{{formatMessage (intlGet "patients.patient.sidebar.sidebarViews.sex") sex=sex}}`,
   },
-];
-
-const InfoView = CollectionView.extend({
-  className: 'patient-sidebar__section',
-  showView(type, text) {
-    /* istanbul ignore next: prevents "null" string */
-    if (!text) text = '';
-
-    const view = new View({
-      className: `patient-sidebar__${ type }`,
-      template: _.constant(_.trim(text)),
-    });
-
-    this.addChildView(view);
-  },
-  onRender() {
-    _.each(patientSet, this.showItem, this);
-
-    this.showPatientFields();
-  },
-  showItem({ heading, data = _.constant(this.model.attributes), template }) {
-    this.showView('heading', heading);
-    this.showView('item', renderTemplate(template, data(this.model)));
+  divider: {
```

Comment:
I think because, even though it doesn't need it, it falls into the `_.extend({ model }, widget)` case of `getWidget` we need to keep the `template` key for this.

Reply:
ah true.